app/models/peep.rb

```diff
@@ -0,0 +1,34 @@
+class Peep
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :content, Text, :required => true
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)